packages/fx-core/src/common/local/localEnvManager.ts

```diff
@@ -59,9 +62,11 @@ export class LocalEnvManager {
 
     // NodeJS
     if (isSPFx) {
-      depsTypes.push(DepsType.SpfxNode);
-    } else if (includeBackend || includeFuncHostedBot) {
-      depsTypes.push(DepsType.FunctionNode);
+      if ((await getAppSPFxVersion(projectPath))?.startsWith("1.16.0")) {
```

Comment:
Does it mean the supported version in future is 1.16.0-x instead of 1.16.x?

Reply:
@HuihuiWu-Microsoft  As we discussed offline, after SPFx 1.16.0 GA, Teams Toolkit will only support node.js 16. The old SPFx node checker will be retired.

Reply:
Yes, for this release, we'll support SPFx v1.16.0-beta.1

Reply:
@XiaofuHuang Right, once SPFx v1.16.0 released, we'll only support SPFx v1.16.0(Node.js v16 is supported by SPFx v1.16.0) and there'll be only one node checker for SPFx